### PR TITLE
feat: revert telemetry server changes

### PR DESCRIPTION
**Pull request by skosito**
# Description

Revert handlers in ts on port 8123. It was replaced by metrics in this PR: https://github.com/zeta-chain/node/pull/1885
In this PR it is reverted, but metrics are still updated. Also changed last scanned block number to have chainName as label same as metrics.

Closes: #2324 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Include instructions and any relevant details so others can reproduce. 

- [ ] Tested CCTX in localnet
- [ ] Tested in development environment
- [ ] Go unit tests
- [ ] Go integration tests
- [ ] Tested via GitHub Actions 

# Checklist:

- [ ] I have added unit tests that prove my fix feature works

### CODE REVIEW

**Comment by brewmaster012** [comment]
has this been tested in localnet?

File: zetaclient/metrics/telemetry.go [hunk 1]
```diff
@@ -11,23 +12,33 @@ import (
 	"github.com/gorilla/mux"
 	"github.com/rs/zerolog"
 	"github.com/rs/zerolog/log"
+
+	"github.com/zeta-chain/zetacore/pkg/chains"
+	"github.com/zeta-chain/zetacore/pkg/constant"
+	"github.com/zeta-chain/zetacore/zetaclient/types"
 )
 
 // TelemetryServer provide http endpoint for Tss server
```
Comment (kingpinXD) [inline zetaclient/metrics/telemetry.go:21]:
```suggestion
// TelemetryServer provides http endpoint for Tss server
```

File: zetaclient/metrics/telemetry.go [hunk 2]
```diff
@@ -11,23 +12,33 @@ import (
...
-	logger         zerolog.Logger
-	s              *http.Server
-	p2pid          string
-	mu             sync.Mutex
-	ipAddress      string
-	HotKeyBurnRate *BurnRate
+	logger                 zerolog.Logger
+	s                      *http.Server
+	p2pid                  string
+	lastScannedBlockNumber map[string]uint64 // chainName => block number
```
Comment (kingpinXD) [inline zetaclient/metrics/telemetry.go:26]: Should we use chainID -> block number instead ?

File: zetaclient/metrics/telemetry.go [hunk 3]
```diff
@@ -67,6 +78,45 @@ func (t *TelemetryServer) GetIPAddress() string {
...
+	t.mu.Unlock()
+}
+
+func (t *TelemetryServer) GetCoreBlockNumber() int64 {
+	t.mu.Lock()
+	defer t.mu.Unlock()
+	return t.lastCoreBlockNumber
+}
+
+func (t *TelemetryServer) SetNumberOfUTXOs(numberOfUTXOs int) {
```
Comment (kingpinXD) [inline zetaclient/metrics/telemetry.go:113]: NIT : maybe we can add comments ,for the functions. 
The functions itself are pretty straightforward but it wont hurt to add a comment for each ?

File: zetaclient/metrics/telemetry.go [hunk 4]
```diff
@@ -131,6 +186,54 @@ func (t *TelemetryServer) ipHandler(w http.ResponseWriter, _ *http.Request) {
 	fmt.Fprintf(w, "%s", t.ipAddress)
 }
 
+func (t *TelemetryServer) lastScannedBlockHandler(w http.ResponseWriter, _ *http.Request) {
+	//w.WriteHeader(http.StatusOK)
```
Comment (kingpinXD) [inline zetaclient/metrics/telemetry.go:190]: Can we remove this line if not needed ?

File: zetaclient/metrics/telemetry.go [hunk 5]
```diff
@@ -131,6 +186,54 @@ func (t *TelemetryServer) ipHandler(w http.ResponseWriter, _ *http.Request) {
...
+
+func (t *TelemetryServer) statusHandler(w http.ResponseWriter, _ *http.Request) {
+	w.WriteHeader(http.StatusOK)
+	t.mu.Lock()
+	defer t.mu.Unlock()
+	s, err := json.MarshalIndent(t.status, "", "\t")
+	if err != nil {
+		t.logger.Error().Err(err).Msg("Failed to marshal status")
+	}
+	fmt.Fprintf(w, "%s", s)
```
Comment (kingpinXD) [inline zetaclient/metrics/telemetry.go:222]: Should we handle the error for Fprintf?

**Review by kingpinXD — APPROVED** [review]
LGTM . 
Minor comments

**Review by lumtis — APPROVED** [review]
LGTM

**Pull request merged**